Update README.md (#85)
amadeus4dev-examples/amadeus-android: README.md

Amadeus has a rich set of APIs for the travel industry. For more details, check out the [Amadeus for Developers Portal](https://developers.amadeus.com).

```diff 
! important

> ! This SDK is being supported by community only, Amadeus for Developers team doesn't support and maintain it.

Reviewer: ```suggestion
! This SDK is maintained by the users’ community only. The Amadeus for Developers team doesn't support and maintain it.
```
